lib/success.js

```diff
@@ -8,11 +8,15 @@ const truncate = require('./truncate')
 const MAX_LENGTH = 2900
 
 module.exports = async (pluginConfig, context) => {
-	const { logger, nextRelease, options } = context
+	const {
+		logger,
+		nextRelease,
+		options,
+		env: { SEMANTIC_RELEASE_PACKAGE, npm_package_name }
+	} = context
 	const { slackWebhook = process.env.SLACK_WEBHOOK } = pluginConfig
 
-	let package_name = context.env.SEMANTIC_RELEASE_PACKAGE
-	if (!package_name) package_name = context.env.npm_package_name
+	let package_name = SEMANTIC_RELEASE_PACKAGE || npm_package_name
```

Comment:
Would it be possible to use `const` here instead?

Reply:
Yes! I'll fix. 👍